packages/esm-patient-chart-app/src/patient-flags/patient-flags-highlight-bar.scss

```
@@ -0,0 +1,39 @@
+@use '@carbon/styles/scss/spacing';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.flagSummary {
+    padding: spacing.$spacing-03;
+    border-bottom: 0.063rem solid $ui-03;
+}
+
+.flagIcon{
+    font-size: medium;
+}
+
+.flagsHighlightTag {
+  padding: 0 0.5rem;
+  border: none;
+  span {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .flagText {
+    margin-left: 0.5rem;
+  }
+}
+
+.highlightBarContainer {
+  display: flex;
+  align-items: center;
+  position: 'relative';
+  padding: 0.5rem;
+  border-bottom: 0.065rem solid $ui-03;
```

```suggestion
  border-bottom: 1px solid $ui-03;
```

1px is better than 0.065 because this becomes very small in case the screen size changes and border gets removed